src/applications/disability-benefits/all-claims/utils/form0781.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Checks if the modern 0781 flow should be shown if the flipper is active for this veteran
+ *
+ * @returns true if set, false otherwise
+ */
+export function showForm0781Pages(formData) {
+  return formData?.syncModern0781Flow === true;
```

Comment:
In addition to evaluating this value, the claim should also be new and include a new disability.

Reply:
claim being new will be handled on the backend. as far as a new disability.. if it's a new claim, wouldn't that be automatic?

Reply:
updated my version to do the same as yours. only difference is the name and location of the function. per our convo I'm punting refactoring your code to use my name spaces until [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/98273)